src/amuse/community/interface/se.py

```diff
@@ -922,7 +922,7 @@ def get_density_profile(self, indices_of_the_stars, number_of_zones = None):
         indices_of_the_stars = self._check_number_of_indices(indices_of_the_stars, action_string = "Querying density profiles")
         if number_of_zones is None:
             number_of_zones = self.get_number_of_zones(indices_of_the_stars)
-        return self.get_density_at_zone([indices_of_the_stars]*number_of_zones, range(number_of_zones) | units.none)
+        return self.get_density_at_zone([indices_of_the_stars]*number_of_zones, list(range(number_of_zones)) | units.none)
```

Comment:
what does a iterator + unit do? (generate the list for the quantity?)

Reply:
list + unit gives a VectorQuantity. Here, the list is very important:
```
In [1]: from amuse.units import units                                                                                                                     

In [2]: a = list(range(4)) | units.m                                                                                                                      

In [3]: a                                                                                                                                                 
Out[3]: quantity<[0, 1, 2, 3] m>

In [4]: b = range(4) | units.m                                                                                                                            

In [5]: b                                                                                                                                                 
Out[5]: quantity<range(0, 4) m>
```

Reply:
So b is a very weird ScalarQuantity in this example

Reply:
Issue #484 and #488